File: Linear_search/Swift/linear_search.swift
let arr = [412, 78, 12, 423, 1]
let target = 1

var found = false
var index = -1

for i in 0..<arr.count {
    if arr[i] == target {
        found = true
        index = i
        break
    }
}

if found {
    print("The element \(target) was found at position \(index)")
} else {
    print("The element  \(target) was not found")
}

File: Linear_search/Rust/linear_search.rs
fn linear_search(arr: &[i32], target: i32) -> Option<usize> {
    for (index, &value) in arr.iter().enumerate() {
        if value == target {
            return Some(index);
        }
    }
    None
}
fn main() {
    let num = [145, 127678, 12, 7784, 101];
    let required = 7784;

    match linear_search(&num, required) {
        Some(index) => println!("The element {} found at position {}", required, index),
        None => println!("The element {} was not found", required),
    }
}